v3/plugins/ms-python/python/2019.2.5558/meta.yaml

```diff
@@ -16,4 +16,4 @@ spec:
       name: "vscode-python"
       memoryLimit: '512Mi'
   extensions:
-    - https://marketplace.visualstudio.com/_apis/public/gallery/publishers/ms-python/vsextensions/python/2019.2.5433/vspackage
+    - https://github.com/microsoft/vscode-python/releases/download/2019.2.5558/ms-python-release.vsix
```

Comment:
Trailing newline

Reply:
I think you mean no newline at end of line. Might be my github settings that converted from 'nix to non-'nix or something.

Reply:
> Worth noting that these changes will break any workspaces using this registry and the old 2019.2.5433 version explicitly.

Yes, but do we want compliance, or compatibility? :)